src/main/java/marquez/common/models/DatasourceUrn.java

```diff
@@ -17,8 +17,10 @@
 import static marquez.common.Preconditions.checkNotBlank;
 
 import com.fasterxml.jackson.annotation.JsonCreator;
+import lombok.EqualsAndHashCode;
 import lombok.NonNull;
 
+@EqualsAndHashCode(callSuper = true)
```

Comment:
We'll need to be careful on merges. Looks like `@EqualsAndHashCode` has been re-introduced. Mind comparing you `DatasourceUrn.java` with what's in master?

Reply:
That's the only difference. I thought to keep it in during the merge b/c I thought that the equality tests (which had failed without it) might still need it. Turns out this is not the case. No other differences WRT master between this file, nor `DatasetUrn.java`.